src/components/Comments/Comment.js

```diff
@@ -418,11 +435,10 @@ const styles = StyleSheet.create({
     flex: 1
   },
   btnReplyText: {
-    fontFamily: fonts.inter[400],
-    fontSize: 13,
-    color: COLORS.balance_gray,
-    marginLeft: 8.98,
-    marginRight: 14
+    fontFamily: fonts.inter[500],
+    fontSize: 12,
+    color: COLORS.gray400,
+    marginLeft: 6
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style for `btnReplyText` has been updated. Make sure this change does not affect the layout in an undesired way.
```diff
+     fontFamily: fonts.inter[500],
+     fontSize: 12,
+     color: COLORS.gray400,
+     marginLeft: 6
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->